data/json/item_groups.json

```diff
@@ -583,6 +583,7 @@
       ["soap", 70],
       ["detergent", 50],
       ["dry_beans", 40],
+      ["dry_lentils", 30],
```

Comment:
This and the other similar occurences need to be formatted [ "dry_lentils", 30 ], to pass Jenkins. It's a format standard thing. Don't worry about the outdated formatting on the other entries in the file, Jenkins will ignore them.

Reply:
The JSON linter can be found [here](http://dev.narc.ro/cataclysm/format.html).

Reply:
It can be, but this file should not be run through the linter.

Reply:
Fixed the formatting.